pkgs/development/tools/gertty/default.nix

```diff
@@ -0,0 +1,48 @@
+{ python3, python3Packages, stdenv }:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "gertty";
+  version = "1.6.0";
+
+  nativeBuildInputs = with python3Packages; [
+    pbr
+    voluptuous
+    dateutil
+    requests
+    urwid
+    ply
+    ordereddict
+    alembic
+    GitPython
+    pyyaml
+  ];
+
+  # XXX Looks like all the buildInputs needs to be propagated...
+  propagatedBuildInputs = with python3.pkgs; [
+    pbr
+    voluptuous
+    dateutil
+    requests
+    urwid
+    ply
+    ordereddict
+    alembic
+    GitPython
+    pyyaml
+  ];
+
+  src = python3.pkgs.fetchPypi {
+    inherit pname version;
+    sha256 = "1pvpca59z5rcir7lz2yimwiqjvd8yhnwjskxna2bslyfwv996d8w";
+  };
+
+  doCheck = true;
+
+  # XXX Missing maintainers?
```

Comment:
According to the contributions guidelines newly submitted packages should have a maintainer: https://nixos.org/nixpkgs/manual/#reviewing-contributions-package-updates
Feel free to add yourself as maintainer in maintainers/maintainer-list.nix. This would mainly involve you receiving notifications on pull-requests about your package or (if these will be re-enabled) e-mail notifications on failing [hydra builds](https://hydra.nixos.org/job/nixpkgs).

Reply:
I'm on the list :)